tools/tertiary-analysis/sc3/sc3-sc3-calc-biology.xml

```diff
@@ -1,17 +1,18 @@
-<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="0.1.0">
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="@TOOL_VERSION@+galaxy0">
```

Comment:
@TOOL_VERSION@ needs to be defined somewhere.

Reply:
possibly in the macro I guess, please see @nh3 scanpy's or the general reference.

Reply:
Please fix the @TOOL_VERSION@ issue in all the other scater wrappers.

Reply:
@cartal I am doing some small editing in your PR. Hopefully it's ok. Please do a `git pull` if you find conflicts when pushing your edits. Thanks!

Reply:
Thanks @nh3